docs/topics/process.rst

```diff
@@ -433,7 +433,7 @@ by setting an ``Order`` as paid.
 
         @staticmethod
         def policy(repository, event):
-            if isinstance(event, Command.Created):
+            if isinstance(event, CreateNewOrder.Created):
```

Comment:
And then maybe the next two lines are now useless and can be removed?

Reply:
Right, but only if `CreateNewOrder` has overriden `Created` event. If it is not overriden then `CreateNewOrder.Created` is still a `Command.Created`

Reply:
Good point, and I like the change. I suppose it would actually work without subclassing the `Created` event, I haven't checked.

Reply:
Yes it would, but also any other event. If we wont subclass and pass `CancelOrder.Created` for ex. it would resolve `isinstance(event, CreateNewOrder.Created)` as true

Reply:
Sure :)